src/commands/package-and-push-with-abs.yaml

```diff
@@ -0,0 +1,95 @@
+parameters:
```

Comment:
Why we shouldn't' always package and push that way?

Reply:
For now I would like to keep the default of packaging with `architect` because `app-build-suite` is still in heavy development and will gain some more features.

We can switch the executor to `app-build-suite` once we're confident it will work on every chart we have.

Reply:
Okay. Not blocking here.